### ISSUE TITLE
WordPress app: Enable REST API

### ISSUE THREAD

**Issue by fluiddot** [wordpress-mobile/WordPress-iOS#23264]
The WP app only supports communicating with the site using XMLRPC. Following a negative app review seems that XMLRPC could be a concerning point when using the app due to security reasons.

> Cool app, and useful functionality for doing limited editing on the run, but XMLRPC is wildly insecure and this app needs two factor authentication for me to use it securely. Will upgrade this review when it has this functionality!

### Expected behavior
The WP app connects to sites using REST API.

### Actual behavior
The WP app connects to sites using XMLRPC.

### Steps to reproduce the behavior
N/A

##### Tested on iPhone 11, iOS 17.0.2, WordPress iOS 25.0

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by av-feaster** [comment]
can you tell me where are the api endpoints to be updated?

**Comment by jkmassel** [comment]
This issue is like 17 projects in a trenchcoat. 

tl;dr – we're doing it! But I'm going to close this issue.
